paper.tex

```diff
@@ -109,11 +106,11 @@ \subsection{Detector models}
 \section{Method}
 To test and evaluate the effect of different data augmentation techniques on the detection capabilities of fake image or text detectors, we created a pipeline of three components: The first component is a generator, which is a model capable of generating fake images or texts. The second component is a processor which augments the data through a specified method, after given a generated image or text. The last component is a detector, which suggests an input being either real or fake. This way we can compare it to the ground truth, to evaluate the capabilities of the detector. By comparing the results with augmented artefacts and with the generated ones without augmentation, we can evaluate the effectiveness of different AI detector-tricking techniques.
 
-We used the aforementioned models for generating and detecting texts and images. As for image processors, one was already given in the description of the "InformatiCup2024" \cite{InformatiCup}, the image noising using the Gaussian noise. Image noise is the random variation of brightness or color information in images produced by the sensor and circuitry of a scanner or digital camera. We used gaussian noice, because it can occure due to natural sources and it is the most common noise that affects images naturally. \cite{mohammed2016study}
+We used some of the aforementioned models for generating and detecting texts and images. For generating we used Stable Diffusion, OpenDalle, GPT2 and Falcon-RW-1B. For detecting we used RoBERTa Base OpenAI Detector, RADAR-Vicuna-7B, AIorNOT and ai-image-det-resnet18. As for image processors, one was already given in the description of the "InformatiCup2024" \cite{InformatiCup}, the image noising using the Gaussian noise. Image noise is the random variation of brightness or color information in images produced by the sensor and circuitry of a scanner or digital camera. We used gaussian noice, because it can occure due to natural sources and it is the most common noise that affects images naturally. \cite{mohammed2016study}
```

Comment:
```suggestion
We used some of the aforementioned models for generating and detecting texts and images. For generating we used Stable Diffusion, OpenDalle, GPT2 and Falcon-RW-1B. The detections were done with the RoBERTa Base OpenAI Detector, RADAR-Vicuna-7B, AIorNOT and ai-image-det-resnet18. As for image processors, one was already given in the description of the "InformatiCup2024" \cite{InformatiCup}, the image noising using the Gaussian noise. Image noise is the random variation of brightness or color information in images produced by the sensor and circuitry of a scanner or digital camera. We used gaussian noice, because it can occure due to natural sources and it is the most common noise that affects images naturally. \cite{mohammed2016study}
```